diyspk/dar/Makefile

```diff
@@ -11,7 +11,7 @@ DISPLAY_NAME = DAR
 STARTABLE = no
 
 # archs without C++11 support are not supported (libthreadar):
-UNSUPPORTED_ARCHS = $(ARM5_ARCHES) hi3535 powerpc ppc824x ppc853x ppc854x
+UNSUPPORTED_ARCHS = $(ALL_ARM5_ARCHS) $(ALL_ARM7L_ARCHS) $(OLD_PPC_ARCHS)
```

Comment:
First I notice this, `ALL_` variables?

Reply:
Sorry, my mistake. Forgot to remove this ...